runtime/syntax/xpm.vim

```diff
@@ -5,6 +5,8 @@
 " Version:	5.4n.1
```

Comment:
Version and last change date need updating too.
Also, the "omit" line in the commit message can be removed now.

Reply:
Will do that when I squash!